Paralayout/UIView+Alignment.swift

```diff
@@ -135,17 +209,14 @@ extension Alignable {
     ) {
         do {
             let receiverView = alignmentContext.view
-            receiverView.untransformedFrame.origin = receiverView.untransformedFrame.origin
-                .offset(
-                    by: try untransformedFrameOffset(
-                        from: position,
-                        to: otherView,
-                        otherPosition,
-                        alignmentBehavior: alignmentBehavior
-                    )
-                )
-                .offset(by: offset)
-                .roundedToPixel(in: receiverView)
+            receiverView.untransformedFrame.origin = try untransformedFrameOrigin(
```

Comment:
There are two ways we could add test coverage here:
1. Add a snapshot test with two identical sibling views, calling `.align()` once on one of them, and twice on the other
2. Add a unit test with a specific configuration that triggers the issue

In either event, we should probably add the test in a distinct PR, merge it first, then rebase this PR against it with the fix.